### PR TITLE
feat: add first draft ExperienceEvent Content field group

### PR DESCRIPTION
**Pull request by ilievlad73**
Adding first draft experience event content field group

### CODE REVIEW

File: components/fieldgroups/experience-event/experienceevent-content.schema.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-content",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Content Information",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Provides information content.",
```
Comment (anandphatak) [inline components/fieldgroups/experience-event/experienceevent-content.schema.json:15]: You can add more details in the description. For example, "Content related information for the content analytics"

Comment (ilievlad73) [reply components/fieldgroups/experience-event/experienceevent-content.schema.json]: Followed some other examples, and now it's something like this: Content analytics data such as eventType, experience ID, and assets IDs.

File: components/fieldgroups/experience-event/experienceevent-content.schema.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            "xdm:eventType": {
+              "title": "Content Event Type",
+              "type": "string",
+              "meta:enum": {
+                "view": "View",
+                "click": "Click"
+              },
+              "description": "The primary content event type."
+            },
+            "xdm:experience": {
```
Comment (anandphatak) [inline components/fieldgroups/experience-event/experienceevent-content.schema.json:33]: I'll recommend changing the field name to `xdm:contentExperience`

File: components/fieldgroups/experience-event/experienceevent-content.schema.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  "description": "Content analytics data such as eventType, experience ID, and assets IDs.",
+  "definitions": {
+    "experienceevent-content": {
+      "properties": {
+        "xdm:content": {
+          "title": "Content",
+          "type": "object",
+          "description": "Information content.",
+          "properties": {
+            "xdm:eventType": {
```
[the remark this answers is not in the document]
Comment (ilievlad73) [reply components/fieldgroups/experience-event/experienceevent-content.schema.json:24]: Yeah, I do need some eventType specific for assets content tracking, which right now should only be 'view' | 'click'. (view assets or click assets).
I would love to update the eventType enum from the time-series behavior to support a few extra eventTypes but I'm not sure of the downstream implications.
In the near future this eventType might also support: 'viewThumbnail'.

[the remark this answers is not in the document]
Comment (ilievlad73) [reply components/fieldgroups/experience-event/experienceevent-content.schema.json:24]: Maybe would be better just to rename this to `xdm:contentEventType` ?

File: components/fieldgroups/experience-event/experienceevent-content.schema.json [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  "title": "Content Information",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Content analytics data such as eventType, experience ID, and assets IDs.",
+  "definitions": {
+    "experienceevent-content": {
+      "properties": {
+        "xdm:content": {
```
[the remark this answers is not in the document]
Comment (ilievlad73) [reply components/fieldgroups/experience-event/experienceevent-content.schema.json:19]: `xdm:digitalContent` would be better?

[the remark this answers is not in the document]
Comment (ilievlad73) [reply components/fieldgroups/experience-event/experienceevent-content.schema.json:19]: In the end, this scope would store assetsIds (image|videos) and some experienceID that would reflect content changes across time.

[the remark this answers is not in the document]
Comment (ilievlad73) [reply components/fieldgroups/experience-event/experienceevent-content.schema.json:19]: I'm not completely satisfied with `xdm:digitalContent`. `xdm:experienceContent` would work?

**Review by anandphatak — APPROVED** [review]

File: components/fieldgroups/experience-event/experienceevent-content.schema.json [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+              "type": "object",
+              "properties": {
+                "xdm:experienceID": {
+                  "title": "Content Experience ID",
+                  "type": "string",
+                  "description": "Content experience identifier."
+                }
+              }
+            },
+            "xdm:assets": {
```
Comment (anandphatak) [inline components/fieldgroups/experience-event/experienceevent-content.schema.json:45]: @ilievlad73 - one more thing:
The `xdm:assets` node has only one property, which is `xdm:assetsIDs`. Do you plan to have additional properties in `xdm:assets` nodes in the future? If yes, then following is the recommended structure:
```suggestion
    "xdm:assets": {
        "title": "Content Assets",
        "description": "Content assets tracking",
        "type": "array",
        "items": {
            "title": "Asset Properties",
            "description": "Additional properties of the assets",
            "type": "object",
            "properties": {
                "xdm:assetID": {
                    "title": "Content Assets ID",
                    "description": "Content assets identifier.",
                    "type": "string"
                }
            }
        }
    }
```    
That way in the structure above, you can add additional properties such as `xdm:assetName` next to `xdm:assetID` property, and keep a room for extending it further in the future.

If you don't plan to have any additional properties, then you should eliminate the `xdm:assets` node and bubble-up the existing `"xdm:assetIDs` node in its place.

Comment (ilievlad73) [reply components/fieldgroups/experience-event/experienceevent-content.schema.json:45]: I will bubble-up the existing "xdm:assetIDs node.

**Review by mjasrotia — APPROVED** [review]

**Pull request merged**